frontend/src/components/course-module/components/task-notes/components/task-note-card/task-note-card.tsx

```diff
@@ -14,25 +18,39 @@ type Props = {
 };
 
 const TaskNoteCard: FC<Props> = ({ note, author, createdAt, status }) => {
+  const camelCaseStatus = changeStringCase({
+    caseType: StringCase.CAMEL_CASE,
+    stringToChange: status ?? '',
+  });
```

Comment:
Why do we need this? looks like status starts with lowercase letter

Reply:
yes, I didn't notice